### PR TITLE
Doc describing directory services hacks

### PR DESCRIPTION
**Pull request by dmohs**

### CODE REVIEW

File: api/doc/2017/gsuite-account-creation.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Automated GSuite Account Creation
+
+Researcher accounts must be Google accounts to interact with FireCloud and they must be created in a custom GSuite domain so that we can enforce two-factor authentication. Therefore, automated account creation means automatically creating a new GSuite user. Automating tasks using Google APIs is usually accomplished via a service account.
+
+The target API in this case is Google's User Admin or Directory API. Google has documentation for how to use a service account in this way [1].
+
+[1] https://developers.google.com/identity/protocols/OAuth2ServiceAccount
+
+According to the documentation, after creating the service account, you can delegate access to certain API scopes using the Admin console [2]. In this case, "Client Name" is the client ID of the service account and the scopes are a comma-delimited list. In fact, if you use the email address of the service account in the Client Name field, it will be replaced by the client ID of the account, so this page does seem to have some knowledge about service accounts generally.
```
Comment (danqrodney) [inline api/doc/2017/gsuite-account-creation.md]: This sounds like a bug in their code or documentation or both. Should I email somebody about this?

Comment (dmohs) [reply api/doc/2017/gsuite-account-creation.md]: The point of this doc is for filing that bug. Josh volunteered already but I'd say go ahead.

Comment (jmandel) [reply api/doc/2017/gsuite-account-creation.md]: @dmohs thanks for writing this up! Quick request: could you include the specific scopes you tried putting into that form before the first (failing) snippet of Java client code?

Comment (jmandel) [reply api/doc/2017/gsuite-account-creation.md]: And in that vein: is there any confirmation that you've assigned a scope to a client? Would be good to capture that too.

Comment (dmohs) [reply api/doc/2017/gsuite-account-creation.md]: Great suggestion, thank you. Done.

**Comment by jmandel** [comment]
Thanks @dmohs -- I created an internal bug at Google (66995384) for this.

**Comment by dmohs** [comment]
Our code is now the most up-to-date place for information, so closing this.

**Pull request closed without merging**